test/notification.js

```diff
@@ -21,9 +21,12 @@ let {
   createSandbox, setupTimerAndXMLHttp, setupRandomResult, unexpectedError, Cr
 } = require("./_common");
 
+
```

Comment:
Nit: Please could you remove this extra newline again (and the one on line 29)? I think the code looks OK without those.